Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -11689,6 +11797,40 @@ end
 					},
 					Notes = "Returns true if the specified player is queued to be transferred to a World.",
 				},
+				RegisterForgeMod =
+				{
+					Params =
+					{
+						{
+							Name = "ModName",
+							Type = "string",
+						},
+						{
+							Name = "ModVersion",
+							Type = "string",
+						},
+					},
+					Notes = "Add a Forge mod name/version to the server ping list.",
```

Comment:
I think the best approach, from the plugin author's perspective, would be to raise a Lua error. In order to do that, though, you'll need to bind the function manually, so that it can do the check itself and raise the error. See the `cLuaState::ApiParamError` for code that raises a Lua error.

With such behavior, a plugin couldn't do much damage if it didn't check for errors, and it could still provide some kind of fallback by using `pcall`:
```lua
local isSuccess = pcall(cServer:Get().RegisterForgeMod, "name", "version")
if not(isSuccess) ...
```